### ISSUE TITLE
examples?

### ISSUE THREAD

**Issue by mvexel** [mvexel/overpass-api-python-wrapper#37]
I started a simple Django example app leveraging the Overpass API wrapper. See https://github.com/mvexel/overpass-api-python-wrapper/tree/example. It would be nice to have some more?

**Comment by mmd-osm** [comment]
Don't want to create an issue on its own, as it's fairly obvious. The following example in README.md lacks some `"` around Highway 51 (otherwise you'll get a syntax error form Overpass API.

```C
>>> way_query = overpass.WayQuery('[name=Highway 51]')
>>> response = api.Get(way_query)
```
should read:

```C
>>> way_query = overpass.WayQuery('[name="Highway 51"]')
>>> response = api.Get(way_query)
```

**Comment by mmd-osm** [comment]
Found this one example request on help osm page:  https://help.openstreetmap.org/questions/47075/overpass-api-python-save-tags

**Comment by mvexel** [comment]
Merged #42 and #44
